easybuild/easyblocks/generic/rpackage.py

```diff
@@ -132,7 +132,10 @@ def make_cmdline_cmd(self, prefix=None):
         else:
             prefix = ''
 
-        loc = self.start_dir or self.ext_dir or self.ext_src
+        if self.start_dir:
+            loc = os.path.join(self.ext_dir or os.path.sep, self.start_dir)
+        else:
+            loc = self.ext_dir or self.ext_src
```

Comment:
As a side effect of the changes in https://github.com/easybuilders/easybuild-framework/pull/3426, this doesn't work as intended anymore...

By the time you reach this code, `self.start_dir` has already been changed to the value of `self.ext_dir`, because the existence check done in `_set_start_dir` on the relative path in `self.start_dir` has failed...

Reply:
This was actually a bug introduced with https://github.com/easybuilders/easybuild-framework/pull/3426, should be fixed with https://github.com/easybuilders/easybuild-framework/pull/3435...